packages/node_modules/@webex/plugin-meetings/src/devices/index.js

```diff
@@ -0,0 +1,134 @@
+/*!
+ * Copyright (c) 2015-2018 Cisco Systems, Inc. See LICENSE file.
+ */
+
+/**
+ *
+ *
+ * @export
+ * @class Devices
+ */
+export default class Devices {
+  /**
+   * Creates an instance of Devices.
+   * @param {*} {
+   *     id, status, isPaired, pairedToken, type
+   *   }
+   * @memberof Devices
+   */
+  constructor({
+    id, status, isPaired, pairedToken, type
+  }) {
+    this.id = id;
+    this.status = status;
+    this.isPaired = isPaired;
+    this.pairedToken = pairedToken;
+    this.type = type;
+  }
+
+  /**
+   *
+   * Request pairing of a device, AKA resource.
+   * @param {string} id
+   * @param {string} token
+   * @returns {Promise}
+   * @memberof Devices
+   */
+  requestPair(id, token) {
+    return new Promise((resolve, reject) => {
+      this.id = id;
+      this.pairedToken = token;
+      resolve(true);
+    });
+  }
+
+  /**
+   * TODO: Can a participant pair with a single device or multiple devices?
+   *
+   * @param {string} pin
+   * @returns {Promise}
+   * @memberof Devices
+   */
+  pair(pin) {
+    return new Promise((resolve, reject) => {
+      // do something with the pin/challenge
+      this.isPaired = true;
+      resolve(true);
+    });
+  }
+
+  /**
+   *
+   * Requires someone to be paired.
+   * @param {string} id
+   * @returns {Promise}
+   * @memberof Devices
+   */
+  join(id) {
+    return new Promise((resolve, reject) => {
+      if (!this.isPaired) reject();
```

Comment:
I'm surprised the linter didn't pick this up. I would move this to two lines for consistency.